src/ui/inspector/details/Properties.vue

```diff
@@ -95,6 +95,7 @@ export default {
             const timestampLabel = this.domainObject.modified ? 'Modified' : 'Created';
             const timestamp = this.domainObject.modified ? this.domainObject.modified : this.domainObject.created;
             const notes = this.domainObject.notes;
+            const version = this.domainObject.version;
```

Comment:
This is nice, now any other objects that have versions will also display them (if we have any).